src/components/Value/CustomValue.tsx

```diff
@@ -20,7 +23,7 @@ const CustomValue: React.FC<NectarCustomValue> = ({
   return (
     <StyledCustomValue as={as} lang={lang}>
       {entries?.map((entry, index) => [
-        index > 0 && ", ",
+        index > 0 && `${delimiter} `,
```

Comment:
Should this be...

```suggestion
        index > 0 && `${delimiter}`,
```
_OR_
```suggestion
        index > 0 && delimiter,
```

to be even more explicit?

Reply:
@mathewjordan Mmmm, not sure.  I did keep the space in here, but you're suggesting the default `delimiter` value would be `", "`?    So if I'm using this, it'd be...

`<Metadata ... customValueDelimiter="*, " />